draft-ietf-httpbis-messaging-latest.xml

```diff
@@ -405,7 +405,8 @@
 <t>
    Intermediaries that process HTTP messages (i.e., all intermediaries
    other than those acting as tunnels) &MUST; send their own HTTP-version
-   in forwarded messages.  In other words, they are not allowed to blindly
+   in forwarded messages, unless it is purposefully downgraded as a workaround
+   for an upstream issue. In other words, they are not allowed to blindly
    forward the <x:ref>start-line</x:ref> without ensuring that the
    protocol version in that message matches a version to which that
    intermediary is conformant for both the receiving and
```

Comment:
This sentence is very long and convoluted.  While you are here, why not just:

> forward the start-line without ensuring that they support the indicated version.  Forwarding an HTTP message without rewriting [...]

(I think that implies both the sending of requests and the receiving of responses amply.)